charts/kyverno/Chart.yaml

```diff
@@ -9,15 +9,22 @@ keywords:
   - kubernetes
   - nirmata
   - policy agent
+  - policy
   - validating webhook
-  - admissions controller
+  - admission controller
+  - mutation
+  - mutate
+  - validate
+  - generate
+  - supply chain
+  - security
 home: https://kyverno.io/
 sources:
   - https://github.com/kyverno/kyverno
 maintainers:
-  - name: Nirmata
-    url: https://kyverno.io/
-kubeVersion: ">=1.16.0-0"
+  - name: kyverno-maintainers
+    email: cncf-kyverno-maintainers@lists.cncf.io
+kubeVersion: ">=1.25.0-0"
```

Comment:
Why `1.25`? What changed between `1.10.x` and `1.11.x` to drop compatibility with `1.24` as per the [compatibility matrix](https://kyverno.io/docs/installation/#compatibility-matrix)?

Reply:
@rbjorklin We guarantee compatibility of Kyverno current release with the 4 latest releases of kubernetes. Older versions may still work but they are not tested.

cc @realshuting @chipzoller

Reply:
Alright. The reason I asked is because trying to run `helm template --kube-version 1.24.13 ...` fails with the following due to this change: `Error: chart requires kubeVersion: >=1.25.0-0 which is incompatible with Kubernetes v1.24.13`

Reply:
You will have to update your command. The fact is we should be modifying the min version with every minor release but we haven't been. Now we are. This is a change that was due for some time.

Reply:
I'll try lying to `helm` to circumvent the check and see if it still works. Thanks for responding! :heart:

Reply:
Logged this issue to automatically bump the version upon new releases.